doc/cffi-manual.texinfo

```diff
@@ -6983,13 +6983,22 @@ invocation.
 @node Static Linking, Limitations, The Groveller, Top
 @chapter Static Linking
 
-On recent enough versions of supported implementations (currently,
-GNU CLISP 2.49, CMUCL 2015-11, and SBCL 1.2.17), and with a recent
-enough ASDF (3.1.2 or later), you can create a statically linked
-Lisp executable image that includes all the C extensions
-(wrappers and any other objects output by @code{compile-op})
-as well as your Lisp code --- or a standalone application executable.
-This makes it easier to deliver your code as a single file.
+@impnote{``Static Linking'' here does not mean the same as in the C
+world. It does not build an ELF static binary. It builds an image or
+program where all the libraries built by CFFI (grovel or wrapper
+files) are statically embedded in the image or program. The resulting
+binary is still typically an ELF shared binary, relying on various
+shared libraries such as libc. Additionally, system foreign libraries
+are not embedded either.}
+
+On recent enough versions of supported implementations (currently, GNU
+CLISP 2.49, CMUCL 2015-11, and SBCL 1.2.17, but not SBCL 2.0.3), and
```

Comment:
what happened with SBCL 2.0.3?

Reply:
This is fixed since https://github.com/sbcl/sbcl/commit/b399b8f390ca9cf6e0684d08f005b6e55aab7155

But it was appending contrib/ to sbcl-homedir-pathname, which means cffi-toolchain couldn't find the sbcl.mk file.

Reply:
Wait, that's 2.0.4 isn't it...

Reply:
Fixed.